src/Symfony/Component/Scheduler/Tests/Trigger/PeriodicalTriggerTest.php

```diff
@@ -108,7 +108,7 @@ public static function provideForToString()
     /**
      * @dataProvider providerGetNextRunDates
      */
-    public function testGetNextRunDates(\DateTimeImmutable $from, TriggerInterface $trigger, array $expected, int $count = 0)
+    public function testGetNextRunDates(\DateTimeImmutable $from, TriggerInterface $trigger, array $expected, ?int $count = null)
```

Comment:
we never pass null either indeed:
```suggestion
    public function testGetNextRunDates(\DateTimeImmutable $from, TriggerInterface $trigger, array $expected, int $count)
```

and let's remove the `?? count below`